### PR TITLE
chore: make convert/build work with cuedb

### PR DESCRIPTION
**Pull request by rawkode**
This isn't perfect,but it's a lot of fixes for cuedb support

1. **Assumption:** Removed `body_raw`, using `body`: I wasn't sure why we made a distinction?

2. Commented out "new" command until we have embedded Cue templates

3. **Assumption:** init didn't need "base", as it's the current working directory.

4. **Break:** BuildDir directories aren't created automatically, need to make this work with  new tables approach

### CODE REVIEW

File: blox/profile.cue [hunk 1]
```diff
@@ -11,15 +11,18 @@
...
 
 		url: string
 		profile_id?: string
+		body?: string
 	}
 
 	#Profile: {
 		_model: {
 			plural: "profiles"
+			supportedExtensions: ["yaml", "yml", "md", "mdx"]
```
Comment (bketelsen) [inline blox/profile.cue:25]: can you add .svx to all the content types that support markdown please? It's svelte's version of mdx.

File: cuedb/schema.go [hunk 2]
```diff
@@ -110,48 +111,46 @@ func GetSchemaV1Metadata(schema cue.Value) (SchemaV1Metadata, error) {
 
 // V1_MODEL is the cue representation of
 // Model metadata
+type ModelV1Metadata struct {
+	Plural              string
+	SupportedExtensions []string
+}
+
 const V1_MODEL = `
```
Comment (bketelsen) [inline cuedb/schema.go:119]: golint is going to complain about ALL_CAPS. Go style is CamelCase even for constants.

**Review by bketelsen — APPROVED** [review]
Looks good so far.

File: blox/profile.cue (hunk 1 above)
Comment (rawkode) [reply blox/profile.cue:25]: No

Comment (rawkode) [reply blox/profile.cue:25]: Sure, I'll add these extensions to the global model's default

**Pull request merged**